.github/workflows/validate-repackaged-binary-sizes.yml

```diff
@@ -0,0 +1,88 @@
+name: Validate manywheel binaries
+
+# This workflow validates the size of the manywheel binaries after repackaging for PyPi
+# Specify the direct URLs to the binaries (from https://download.pytorch.org/whl/test/torch/) in the matrix
+# along with the python version.
+#
+# The workflow will:
+#  * download the binaries,
+#  * run release/pypi/prep_binary_for_pypi.sh
+#  * run smoke tests on the repackaged binaries
+#  * display the size before and after repackaging as the workflow annotation
+#  * optionally upload the repackaged binaries as artifacts (for debug or promotion)
+
+on:
+  pull_request:
```

Comment:
Perhaps we should add on request, when running manually.
And add a pytorch version and cuda version as a parameter. Since we do plan on introducing cu118 soon.
Also add nightly/test as a parameter as well